bundles/InstallBundle/src/Command/InstallCommand.php

```diff
@@ -32,6 +32,8 @@
 use Symfony\Component\Console\Output\BufferedOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use function explode;
+use function implode;
```

Comment:
Accidental imports?

Reply:
No, they were added automatically by PhpStorm, since they are used further down in the code.

Reply:
Yep, that's true - but in newer bundles we do already 😊